apps/website/docs/components/skeleton.mdx

```diff
@@ -0,0 +1,28 @@
+---
+Title: Skeleton
+description: A visual placeholder for an element that is loading.
+---
+
+A visual placeholder for an element that is loading.
+
+<Demo fileName="skeleton-card.tsx" />
+
+## Best Practices
+
+**When to use**
+
+When you need to prevent Cumulative Layout Shift (CLS) or provide a visual placeholder for an
+element that is loading.
+
+## Content
+
+## Accessibility
```

Comment:
lets remove best pracitces, content and accessibility for now. Follow this pattern:

<title>
<description>
<demo>
Examples
<examples>
Props
<props_table>

Reply:
> lets remove best pracitces, content and accessibility for now. Follow this pattern:
> 
> <title> Examples Props

Removed those headers. Unsure what you mean by `<title> Examples Props` pattern however. Take a look now to see if I should make additional updates!